config/impl-base/src/main/java/org/jboss/arquillian/config/impl/extension/ConfigExtension.java

```diff
@@ -29,5 +30,8 @@
     @Override
     public void register(ExtensionBuilder builder) {
         builder.observer(ConfigurationRegistrar.class);
+
+        builder.service(ConfigurationPlaceholderResolver.class, ConfigurationSysPropResolver.class);
+        builder.service(ConfigurationPlaceholderResolver.class, ClasspathConfigurationPlaceholderResolver.class);
```

Comment:
Please, unify the naming template of the classes:
`ConfigurationSysPropResolver`
`ClasspathConfigurationPlaceholderResolver`